src/Router/Boot.tsx

```diff
@@ -3,40 +3,35 @@ import React from "react"
 import { GridThemeProvider } from "styled-bootstrap-grid"
 import { GlobalStyles } from "Styleguide/Elements/GlobalStyles"
 import { Grid } from "Styleguide/Elements/Grid"
-import { Provider as StateProvider } from "unstated"
+import { Container, Provider as StateProvider } from "unstated"
 import { ResponsiveProvider } from "Utils/Responsive"
+import { Breakpoint } from "Utils/Responsive"
 import { ContextProvider } from "../Components/Artsy"
-import { AppState } from "./state"
-import { BootProps } from "./types"
+import { AppStateContainer } from "./types"
 
-export const Boot: React.SFC<BootProps> = ({ children, ...props }) => {
-  const appState = new AppState(props)
-
-  return (
-    <StateProvider inject={[appState]}>
-      <ContextProvider
-        relayEnvironment={props.relayEnvironment}
-        currentUser={props.currentUser}
```

Comment:
Neither of these props were being passed in, they were only inside the `props.system` object.